.travis/run.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+set -x
+if [[ "$(uname -s)" == "Darwin" ]]; then
+    firefox-ui-tests --binary /Applications/FirefoxNightly.app/Contents/MacOS/firefox
+else
+    firefox-ui-tests --binary firefox/firefox
+fi
```

Comment:
I'm ok with it for now. Once our CLI command can handle installers (which will be soon) we can refactor this code.